packages/eslint-config-sheriff/src/utils/getTsNamingConventionRule.ts

```diff
@@ -1,58 +1,72 @@
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import type { TSESLint } from '@typescript-eslint/utils';
+
 interface GetTsNamingConventionRuleOptions {
   isTsx: boolean;
+  isAstroEndpoint?: boolean;
 }
 
 export const getTsNamingConventionRule = ({
   isTsx,
-}: GetTsNamingConventionRuleOptions) => {
+  isAstroEndpoint,
+}: GetTsNamingConventionRuleOptions): TSESLint.FlatConfig.Rules => {
+  const options: unknown[] = [
```

Comment:
Why are we specifying the type here? Doesn't `tsc` already infers a better one by itself?

Reply:
No because it doesn't have `filter` on any of these so will get a typescript error when appending below unless I do something like `as never` on the object I'm appending.

Reply:
If you'd prefer I could also just do what you do with `isTsx` e.g. `isAstroEndpoint && { ...` and `.filter(Boolean)`.